packages/grid/_modules_/grid/models/colDef/gridSingleSelectOperators.ts

```diff
@@ -6,13 +6,14 @@ export const getGridSingleSelectOperators: () => GridFilterOperator[] = () => [
   {
     value: 'is',
     getApplyFilterFn: (filterItem: GridFilterItem) => {
-      if (!filterItem.columnField || !filterItem.value || !filterItem.operatorValue) {
```

Comment:
We're already validating if there's a `columnField` and `operatorValue` before calling the filter function. Related to https://github.com/mui-org/material-ui-x/issues/1961#issuecomment-869177167